### ISSUE TITLE
[ADE Support]: VA Notify - Discussion/Training for Understanding Accessibility Testing and ADE Services

### ISSUE THREAD

**Issue by EvanAtCoforma** [department-of-veterans-affairs/va.gov-team#108380]
## Description of the Work

Met with Jake Uhteg regarding their 508 audit that received a multitude of defects and how they can improve their accessibility testing needs for future audits. We discussed how to conduct manual testing, how to improve their current processes, and how they can reach out to the ADE team for more in-depth accessibility testing and preparation prior to their next 508 audit.

### Background

Jake and the VA Notify team have a lot of custom components throughout their product, and while he was conducting high level AT testing, he wasn't able to catch everything he needed to in order for the 508 audit to be successful. He reached out to us to see how they can improve moving forward.

## About the Team 

- **Team name**: VA Notify
- **Product name:** VA Notify app
- **Product manager:**  
- **Slack Channel:** [va-notify-public](https://dsva.slack.com/archives/C010R6AUPHT)

## Supporting Artifacts 
- [Artifact 1](https://docs.google.com/document/d/1rNC1IyXmMoMiMLj4yyAJl1mZ3ZqxU0cio18pEI-mY0Q/edit?tab=t.0#heading=h.jxiu03v0tlie)

## ADE Tasks
- [x] Collaborate with Jake to see how they can improve accessibility processes
- [x] Point VA Notify team to ADE services

### Ticket Updates
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [x] _Label with `Actual` (level of effort it took to complete this ticket)_
- [x] _Update date in_ `Closed Date`
- [x] **Read** Important Notifications (ADE Internal) at the bottom of this ticket.
- [x] **Delete** Important Notifications (ADE Internal) at the bottom of this ticket, after reading.

**Comment by EvanAtCoforma** [comment]
Met with Jake to discuss his options and how he can reach to the ADE team for proper assistance prior to their next 508 audit
